### PR TITLE
Add badges to README

### PR DESCRIPTION
**Pull request by adamnsch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Neo4j Graph Data Science Client
 
+[![Latest version](https://img.shields.io/pypi/v/graphdatascience)](https://pypi.org/project/graphdatascience/)
+[![PyPI downloads month](https://img.shields.io/pypi/dm/graphdatascience)](https://pypi.org/project/graphdatascience/)
+![Python versions](https://img.shields.io/pypi/pyversions/graphdatascience)
```
Comment (soerenreichardt) [inline README.md:5]: Is there a `[` missing?

**Review by soerenreichardt — APPROVED** [review]
🎉

File: README.md (hunk 1 above)
Comment (FlorentinD) [reply README.md]: added one :)

Comment (adamnsch) [reply README.md]: No, that's only if you want to add a link to the badge. It's wrong now :see_no_evil: I will push a fix to main

**Pull request merged**